bool usbSwitch1
bool usbSwitch2
